ts/screens/messages/MessagesScreen.tsx

```diff
@@ -32,10 +33,24 @@ export type OwnProps = Readonly<{
 }>;
 
 export type IMessageDetails = Readonly<{
-  item: Readonly<MessageWithContentPO>;
+  item: Readonly<IMessageWithPaymentData>;
   index: number;
 }>;
 
+export type IPaymentData = Readonly<{
+  amount: number;
```

Comment:
@LsKoder @fgiobergia  what type should we use for `amount` ?

Reply:
I suggest integer (eurocents) everywhere:
https://github.com/teamdigitale/italia-ts-commons/blob/master/src/numbers.ts#L110